staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go

```diff
@@ -197,7 +197,10 @@ func (s *specAggregator) updateServiceLocked(name string) error {
 		if result.Err != nil {
 			return result
 		}
-		return cached.NewResultOK(aggregator.FilterSpecByPathsWithoutSideEffects(result.Data, []string{"/apis/"}), result.Etag)
+		group := specInfo.apiService.Spec.Group
+		version := specInfo.apiService.Spec.Version
+
+		return cached.NewResultOK(aggregator.FilterSpecByPathsWithoutSideEffects(result.Data, []string{"/apis/" + group + "/" + version}), result.Etag)
```

Comment:
question on the trailing slash behavior at https://github.com/kubernetes/kubernetes/pull/123624#discussion_r1513838350